Documentation/rfc/v3api.proto

```diff
@@ -32,24 +43,11 @@ service KV {
   // LeaseRevoke revokes a lease. All the key attached to the lease will be expired and deleted.
   rpc LeaseRevoke(LeaseRevokeRequest) returns (LeaseRevokeResponse) {}
 
-  // LeaseAttach attaches keys with a lease.
-  rpc LeaseAttach(LeaseAttachRequest) returns (LeaseAttachResponse) {}
```

Comment:
Will `Put` supersede `LeaseAttach`'s functionality because of the KV lease field? If so, would this mean that attaching a lease would be a read-modify-write (e.g., I must know the key's value before I can do the PUT to attach the lease)?